doc/README_Source.md

```diff
@@ -47,8 +46,7 @@ It is possible to have the Source connector output CSV string by using StringCon
 ## Quick Start
 
 ### Prerequisites
-* [Confluent Platform](https://docs.confluent.io/current/installation/index.html#installation-overview)
```

Comment:
i like the addition of the docs about EventHubs, but why are you removing the docs about installation of the Confluent platform? Did you find they are not relevant, unhelpful, confusing, incomplete?

Reply:
Hmm, for a quickstart I thought it was confusing, since we don't need the Confluent Platform at all. But very little change is necessary to make it work with the Confluent Platform. Maybe just document the necessary changes?

Reply:
true. it is much quicker to get going now that we have support for the EventHub Kafka endpoint.

Reply:
I will document the steps needed to run if you don't have EventHubs configured.